### DISCUSSION TITLE
Is anyone monitoring the paid support help?

### DISCUSSION THREAD

**Discussion by nCubed** [forwardemail/forwardemail.net#252] [General]
As the title states, is anyone monitoring the paid support help? I submitted a support request last week and have not heard from anyone. Have also seen reports on Reddit of other paid users not receiving any replies. Other than the [help page form](https://forwardemail.net/en/help), is there any other way to request support without posting public information here?

**Comment by titanism** [comment]
@nCubed we usually reply to every help request within a few hours during business days, sometimes longer on weekends - if you directly email `support@forwardemail.net` or follow up on your ticket # over email we can take another look, perhaps you didn't get a reply from us!

**Comment by graemev** [comment]
I've had delays of several weeks getting a response. One big issue is the lack of any ticket tracking system ...so I get get a number, then another, then another ...2 weeks later I get a response ... need to dig back tough requests .

**Reply by titanism** [reply]
Yes we monitor all support requests and answer them in priority and order they are received.  We'll take a look for you right now too.

Note we're having an issue with our IMAP/POP3 server which should be resolved soon.  <https://github.com/forwardemail/status.forwardemail.net/issues/1165>
